CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ## Changed
 - [Aptly] Publishing is now conditionned on existing published repositories instead of locals
 - [Apt] Remove explicit architecture on Manala repository pattern
+- [Apt] Trust MySQL repository on stretch
 ## Added
 - [Aptly] `Repositories` task now handle `architecture` when publishing
```

Comment:
should not be here (copy/paste mistake probably)

Reply:
fixed on master